models/articles.modles.js

```diff
@@ -1,5 +1,34 @@
 const db = require('../db/connection');
 
+exports.selectArticles = () => {
+  const queryStr = `SELECT 
+  articles.author, 
+  articles.title, 
+  articles.article_id, 
+  articles.topic, 
+  articles.created_at, 
+  articles.votes, 
+  articles.article_img_url,
+  COALESCE(COUNT(comments.comment_id), 0)::int as comments_count
+  FROM articles
+  LEFT JOIN comments ON articles.article_id = comments.article_id
+  GROUP BY 
+  articles.author, 
+  articles.title, 
+  articles.article_id, 
+  articles.topic, 
+  articles.created_at,
+  articles.votes, 
+  articles.article_img_url
+  ORDER BY articles.created_at DESC;
```

Comment:
I'm not 100% on this but pretty sure you can GROUP BY articles.article_id alone

Reply:
That works, it didnt in the query with coalesence for some reason